### PR TITLE
chore: consistent file casing

### PR DESCRIPTION
**Pull request by talkor**
All component-related files are PascalCased now

https://monday.monday.com/boards/3532714909/pulses/6714025950

### CODE REVIEW

File: packages/core/src/components/AlertBanner/__tests__/AlertBanner-snapshots.test.js
Comment (talkor) [inline packages/core/src/components/AlertBanner/__tests__/AlertBanner-snapshots.test.js:1]: United with another file with a similar name that was 90% similar

File: packages/core/src/components/AlertBanner/__tests__/__snapshots__/AlertBanner-snapshots.test.js.snap
Comment (talkor) [inline packages/core/src/components/AlertBanner/__tests__/__snapshots__/AlertBanner-snapshots.test.js.snap:1]: Snapshot changed due to `should` typo

File: packages/core/src/components/AlertBanner/__tests__/__snapshots__/AlertBanner.snapshot.test.js.snap
Comment (talkor) [inline packages/core/src/components/AlertBanner/__tests__/__snapshots__/AlertBanner.snapshot.test.js.snap:1]: Snapshot updated due to the "should" typo

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss
Comment (talkor) [inline packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss:1]: Kept the same convention as the other storybook related components but I'm all for moving them also in the usual convention, but in a different PR

File: packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp
Comment (YossiSaadi) [inline packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp:1]: Shouldn't this have a story? I see it was called temp but I don't see any replacement

File: packages/core/src/components/List/__stories__/list-story.scss
Comment (YossiSaadi) [inline packages/core/src/components/List/__stories__/list-story.scss:1]: just commenting to make sure it was intended - made sure it looks the same?

File: packages/core/src/components/MenuButton/__stories__/menuButton.style.scss
Comment (YossiSaadi) [inline packages/core/src/components/MenuButton/__stories__/menuButton.style.scss:1]: same, was intentionally?

File: packages/core/src/components/Tabs/TabsContext/__stories__/tabsContext.stories.temp
Comment (YossiSaadi) [inline packages/core/src/components/Tabs/TabsContext/__stories__/tabsContext.stories.temp:1]: also has no replacement file

File: packages/core/src/components/Tabs/TabsContext/__stories__/TabsContext.stories.scss
Comment (YossiSaadi) [inline packages/core/src/components/Tabs/TabsContext/__stories__/TabsContext.stories.scss:1]: seem like you removed the story of it, should it still have a stylesheet file if the story was removed on purpose?

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { useMemo } from "react";
 import { RelatedComponent } from "vibe-storybook-components";
 import VirtualizedList from "../../../../../components/VirtualizedList/VirtualizedList";
-import { generateItems } from "../../../../../components/VirtualizedList/__stories__/virtualizedList.stories.helpers";
-import styles from "./virtualizedList.module.scss";
+import { generateItems } from "../../../../../components/VirtualizedList/__stories__/VirtualizedList.stories.helpers";
+import styles from "./virtualized-list.module.scss";
```
Comment (YossiSaadi) [inline packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx:5]: I guess a mistake?

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss
Comment (YossiSaadi) [inline packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss:1]: I guess a mistake?

**Review by YossiSaadi — APPROVED** [review]
Commented on a few stuff I wasn't sure intended, other than that looks cool
also, about constants files, I'm not sure about camelCase versus kebab case, but we can change it later if we'll decide to so don't change it here, let's discuss in next dev meeting

File: packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp
Comment (talkor) [reply packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp:1]: It's an internal component used for the old EditiableHeading, so it didn't have a story

File: packages/core/src/components/List/__stories__/list-story.scss
Comment (talkor) [reply packages/core/src/components/List/__stories__/list-story.scss:1]: It was unused

File: packages/core/src/components/MenuButton/__stories__/menuButton.style.scss
Comment (talkor) [reply packages/core/src/components/MenuButton/__stories__/menuButton.style.scss:1]: Yes, unused

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss
Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list.module.scss:1]: Kept the same convention as the other storybook related components but I'm all for moving them also in the usual convention, but in a different PR

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx (hunk 1 above)
Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx:5]: Kept the same convention as the other storybook related components but I'm all for moving them also in the usual convention, but in a different PR

File: packages/core/src/components/Tabs/TabsContext/__stories__/tabsContext.stories.temp
Comment (talkor) [reply packages/core/src/components/Tabs/TabsContext/__stories__/tabsContext.stories.temp:1]: I don't think we really need a story for that, it was `.temp` for some reason, never was shown in storybook

File: packages/core/src/components/Tabs/TabsContext/__stories__/TabsContext.stories.scss
Comment (talkor) [reply packages/core/src/components/Tabs/TabsContext/__stories__/TabsContext.stories.scss:1]: Good catch, will remove

File: packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp
Comment (YossiSaadi) [reply packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp:1]: @talkor maybe we should add one to the "internal" section

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx (hunk 1 above)
Comment (YossiSaadi) [reply packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx:5]: in other related we have scss with kebab case?

**Review by YossiSaadi — APPROVED** [review]

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx (hunk 1 above)
Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx:5]: Yes, all of them. I don't really like it but I think it's out of scope for now, let's talk about it

File: packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp
Comment (talkor) [reply packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp:1]: Not for this component but in general definitely yeah. I'm removing this component in a different PR

Comment (YossiSaadi) [reply packages/core/src/components/EditableInput/__stories__/editableInput.stories.temp:1]: I only now got that it is the old one

**Pull request merged**